src/main/scala/FileSystem.scala

```diff
@@ -0,0 +1,34 @@
+
+
+class FileSystem {
+  def commandInterpretor() : Unit = {
+    print( "Your cmd (press \"q\" to quit, \"h\" for help): " )
+    val inputKeyboard = getKeyboardInput
+
+    val cmdMatcher = inputKeyboard match{
+      case "q" => "The key q is pressed, the app will close."
+      case "h" =>  help(); commandInterpretor()
+      case "curdir" => println( getCurrentPath ); commandInterpretor()
+      case _   => println("The command is not recognized. Type \"h\" to list the commands and how to use them or \"q\" to quit."); commandInterpretor()
```

Comment:
In Scala, you can use triple quotes so you don't need to escape your inner quotes
```suggestion
      case _   => println("""The command is not recognized. Type "h" to list the commands and how to use them or "q" to quit."""); commandInterpretor()
```